orcid-activemq/src/main/resources/activemq.xml

```diff
@@ -20,18 +20,22 @@
 <!-- this is a spring XML file where we have ActiveMQ Broker embedded -->
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns:broker="http://activemq.apache.org/schema/core"
+       xmlns:broker="http://activemq.apache.org/schema/core"  
+       xmlns:context="http://www.springframework.org/schema/context"     
        xsi:schemaLocation="
        http://activemq.apache.org/schema/core http://activemq.apache.org/schema/core/activemq-core.xsd
+       http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.1.xsd
        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
  
-    <!-- create an ActiveMQ broker -->
+ 	<context:property-placeholder location="${org.orcid.activemq.config.file}" />
```

Comment:
@TomDemeranville In order to deploy this to the servers, we need to be able to configure the properties file path, so, now on, in order to start the server locally, we will need a new property:

-Dorg.orcid.activemq.config.file=classpath:orcid-activemq.properties